cloud/pkg/cloudhub/handler/messagehandler.go

```diff
@@ -425,29 +425,31 @@ func (mh *MessageHandle) MessageWriteLoop(info *model.HubInfo, stopServe chan Ex
 		obj, exist, _ := nodeStore.GetByKey(key.(string))
 		if !exist {
 			klog.Errorf("nodeStore for node %s doesn't exist", info.NodeID)
+			nodeQueue.Done(key)
 			continue
 		}
 		msg := obj.(*beehiveModel.Message)
 
 		if !model.IsToEdge(msg) {
 			klog.Infof("skip only to cloud event for node %s, %s, content %s", info.NodeID, dumpMessageMetadata(msg), msg.Content)
+			nodeQueue.Done(key)
 			continue
 		}
 		klog.V(4).Infof("event to send for node %s, %s, content %s", info.NodeID, dumpMessageMetadata(msg), msg.Content)
 
 		copyMsg := deepcopy(msg)
-		trimMessage(msg)
+		trimMessage(copyMsg)
 
 		for {
 			conn, ok := mh.nodeConns.Load(info.NodeID)
 			if !ok {
 				time.Sleep(time.Second * 2)
 				continue
 			}
-			err := mh.sendMsg(conn.(hubio.CloudHubIO), info, msg, copyMsg, nodeStore)
+			err := mh.sendMsg(conn.(hubio.CloudHubIO), info, copyMsg, msg, nodeStore)
```

Comment:
Also need to modify the parameters name in sendMsg func :)

Reply:
ok,  the parameter name in sendMsg func has been renamed.